.github/workflows/preview.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: make install
 
       - name: Build the website
-        run: make build
+        run: PREVIEW=true make build
```

Comment:
you can pass it the other way around

```make build PREVIEW=true``` :)

Reply:
Are you on a call? :)

Reply:
not anymore :D